datameta/api/metadata.py

```diff
@@ -18,22 +18,25 @@
 from pyramid.view import view_config
 from . import DataHolderBase
 from ..models import MetaDatum
-from .. import resource, security
+from .. import resource, security, siteid
+from ..resource import resource_by_id, get_identifier
+from pyramid.httpexceptions import HTTPNoContent, HTTPForbidden
 
 @dataclass
 class MetaDataResponseElement(DataHolderBase):
     """MetaDataSetResponse container for OpenApi communication"""
     id                      :  dict
     name                    :  str
-    is_mandatory            :  bool
+    isMandatory            :  bool
     order                   :  int
-    is_file                 :  bool
-    is_submission_unique    :  bool
-    is_site_unique          :  bool
-    regex_description       :  Optional[str] = None
-    long_description        :  Optional[str] = None
-    reg_exp                 :  Optional[str] = None
-    date_time_fmt           :  Optional[str] = None
+    isFile                 :  bool
+    isSubmissionUnique    :  bool
+    isSiteUnique          :  bool
+    regexDescription       :  Optional[str] = None
+    longDescription        :  Optional[str] = None
+    example                 :  Optional[str] = None
+    regExp                 :  Optional[str] = None
+    dateTimeFmt           :  Optional[str] = None
```

Comment:
Please revert to snake case, the `DataHolderBase` class takes care of converstion